server/src/main/java/org/apache/cassandra/sidecar/handlers/AbstractHandler.java

```diff
@@ -317,4 +324,32 @@ public static String extractHostAddressWithoutPort(HttpServerRequest request) th
         }
         return host;
     }
+
+    /**
+     * Handles the submission and execution of an operational job.
+     *
+     * @param jobManager the manager responsible for submitting and tracking operational jobs
+     * @param config     the service configuration containing execution parameters
+     * @param context    the routing context for the HTTP request/response
+     * @param job        the operational job to be executed
+     */
+    protected void handleOperationalJob(OperationalJobManager jobManager, ServiceConfiguration config, RoutingContext context, OperationalJob job)
```

Comment:
Nice abstracting this out. Instead of adding this to `AbstractHandler` it might be better to add a new `AbstractOperationHandler` inject the `OperationalJobManager` and `SidecarConfiguration` to `OperationHandler` instead of passing them through `handleOperationalJob` method.